tox.ini

```diff
@@ -16,12 +16,13 @@ deps =
    isort
    mccabe
    pytest
+   pytest-xdist
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/ {posargs:}
+    python -Wi {envsitepackagesdir}/coverage run -m pytest -n auto {envsitepackagesdir}/pylint/test/ {posargs:}
```

Comment:
If contributors and developers want, they can pass the flags themselves to tox. It does not have to be a "one size fits it all" entry point for running the tests.